auth/auth-server/src/telemetry/helpers.rs

```diff
@@ -225,6 +236,16 @@ pub(crate) async fn record_external_match_metrics(
         warn!("Error recording request metrics: {e}");
     }
 
+    // Record fill ratio metric
+    let requested_quote_amount = order.get_quote_amount(FixedPoint::from_f64_round_down(price));
+    let matched_quote = match match_resp.match_bundle.match_result.direction {
+        OrderSide::Buy => &match_resp.match_bundle.send,
+        OrderSide::Sell => &match_resp.match_bundle.receive,
+    };
```

Comment:
same thing here, can just directly use the `quote_amount` field on the match result